ui/src/app/edge/live/thermometer/thermometer.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component } from '@angular/core';
+import { ChannelAddress, CurrentData, EdgeConfig, Utils } from 'src/app/shared/shared';
+import { AbstractFlatWidget } from 'src/app/shared/genericComponents/flat/abstract-flat-widget';
+import { formatNumber } from '@angular/common';
+
+@Component({
+  selector: 'Thermometer',
+  templateUrl: './thermometer.component.html'
+})
+export class ThermometerComponent extends AbstractFlatWidget {
+
+  public sensors: EdgeConfig.Component[] = null;
```

Comment:
template variables can be protected with angular 14. https://twitter.com/angular/status/1537181658461265920

Strict nullchecks also require this variable to be of a union type "EdgeConfig.Component[] | null"